neuroglancerjsonserver/backend/database.py

```diff
@@ -40,9 +39,17 @@ def kind(self):
 
     @property
     def json_column(self):
-        return 'v1'
+        return "v1"
+
+    @property
+    def bucket_name(self):
```

Comment:
the bucket path should probably be cached in some way. I think its enough to read it upon instantiation and then keep it around. Querying the same row every time a state is read can cause this to be a hot row (e.g. when programmatically reading from the service)

Reply:
made it a property